contribute/documentation/README.md

```diff
@@ -1,7 +1,14 @@
 # Contribute to our documentation
 
-Welcome. We're glad you're here to help make our technical documentation even better.
+Welcome! We're glad you're here.
 
-For our style guide and writing guidelines, see [Writers' Toolkit](https://grafana.com/docs/writers-toolkit/).
+[Writers' Toolkit](https://grafana.com/docs/writers-toolkit/) is an open-source project intended to empower both internal and external contributors to help make our technical documentation even better.  
 
-If you're interested in contributing to the Writers' Toolkit, refer to the [Writers' Toolkit](https://github.com/grafana/writers-toolkit) repository.
+Go to [Writers' Toolkit](https://grafana.com/docs/writers-toolkit/) for the following resources:
+
+- **Contribute** - Learn how can you contribute to Grafana Labs documentation.
+- **Structure** - Learn how to organize and structure our documentation.
+- **Writing and style guide** - Guidelines that cover aspects of content such as style, tooling, workflows, markdown syntax, and more.
+- **Build and review guide** - Instructions for testing, linting, validating, and backporting our documentation.
+
+If you're interested in contributing to our documentation, go to [Writers' Toolkit](https://github.com/grafana/writers-toolkit) repository.
```

Comment:
Missing word? Maybe we should also clarify that we're talking about contributing to the documentation on writing rather than just "our documentation"?

```suggestion
If you're interested in contributing to our Writers' Toolkit documentation, go to the [Writers' Toolkit](https://github.com/grafana/writers-toolkit) repository.
```

Reply:
fixed, but in different way